bin/start-stunnel

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env bash
 
 main() {
+  STUNNEL=$(command -v stunnel4)
+
+  if [ -z $STUNNEL ]; then
+      echo "buildpack=stunnel at=error stunnel not detected! stunnel not supported on heroku-24+"
+      echo "buildpack=stunnel at=error this buildpack is not required for Redis 6+: https://devcenter.heroku.com/articles/heroku-redis#security-and-compliance"
+      exit 1
+  fi
+
```

Comment:
This will never be executed, since the build will have failed before the file was vendored. As such I don't think it's needed?

```suggestion
```